std/format/read.d

```diff
@@ -20,43 +20,44 @@ import std.format.internal.read;
 import std.traits : isSomeString;
 
 /**
-Reads characters from $(REF_ALTTEXT input range, isInputRange, std,range,primitives)
-`r`, converts them according to `fmt`, and writes them to `args`.
+Reads an input range according to a format string and stores the read
+values into its arguments.
+
+The second version of `formattedRead` takes the format string as
+template argument. In this case, it is checked for consistency at
+compile-time.
```

Comment:
Why not put this information as a comment directly above the second function?

Reply:
Is this possible? I'll have a look...

Reply:
I can't see, how this works out well - maybe I do not know something I should. With my attempts, the comment is either not shown at all or the two overloads are split and I need to duplicate the whole docs... (And we did it this way with `format`, `sformat` and so on too.)

Reply:
Hmm... I see. I guess this is fine too.